tests/integration/security/webhook/webhook_test.go

```diff
@@ -56,10 +56,7 @@ func TestWebhookManagement(t *testing.T) {
 	framework.
 		NewTest(t).
 		Run(func(ctx framework.TestContext) {
-			cfg := inst.Settings()
-			if cfg.IsIstiodEnabled() {
-				ctx.Skip("TODO(github.com/istio/istio/issues/20289)")
-			}
+			ctx.Skip("TODO(github.com/istio/istio/issues/20289)")
```

Comment:
do we lose the ability to check for istiod being enabled?

Reply:
I suppose we do, but we also don't have any tests that actually have istiod disabled and I was planning to remove the optional to disable istiod anyways so I figured I would just remove it? wdyt?

Reply:
ok sgtm

Reply:
cc @lei-tang